in-band-manageability/Makefile

```
@@ -15,6 +15,7 @@ else
 endif
 TARBALL_DIR := $(BUILD_DIR)/$(NAME)-$(PKG_VERSION)
 REGISTRY := 080137407410.dkr.ecr.us-west-2.amazonaws.com
+FUZZTIME ?= 10s
```

```suggestion
FUZZ_TIME ?= 10s
```
Please update to `FUZZ_TIME` so that, if running the `make fuzztest-agents` from the top level makefile, it can pick up the default duration set in common.mk

---

Okay, check your open PR #452 . i should use same now.